tests/test_cloudformation/test_cloudformation_stack_crud_boto3.py

```diff
@@ -312,9 +312,24 @@
 def test_create_stack():
     cf_conn = boto3.client("cloudformation", region_name="us-east-1")
     cf_conn.create_stack(StackName="test_stack", TemplateBody=dummy_template_json)
+    stack = cf_conn.describe_stacks()["Stacks"][0]
+    stack.should.have.key("StackName").equal("test_stack")
```

Comment:
Can we add an assertion to the existing test case, `stack.should.have.key("EnableTerminationProtection").equal(False)`?
Just to ensure that the logic is correct when the parameter is not provided

Reply:
Sure! Included now.